src/main/java/com/codeeva/codeeva/entities/Email.java

```diff
@@ -0,0 +1,46 @@
+package com.codeeva.codeeva.entities;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Objects;
+
+@NoArgsConstructor @AllArgsConstructor
+@Getter @Setter
+@Entity
+
+
+public class Email {
+
+    @Id
+    @javax.validation.constraints.Email
+    @Column(unique = true)
```

Comment:
The @Column(unique = true) annotation is redundant because the @Id annotation is already unique.